server/wizard/steps/step_zoom_self_hosted.go

```diff
@@ -0,0 +1,98 @@
+package steps
+
+import (
+	"github.com/pkg/errors"
+
+	pluginapi "github.com/mattermost/mattermost-plugin-api"
+	"github.com/mattermost/mattermost-plugin-api/experimental/flow/steps"
+	"github.com/mattermost/mattermost-server/v6/model"
+
+	"github.com/mattermost/mattermost-plugin-zoom/server/config"
+)
+
+const (
+	stepNameSelfHostedQuestion = "self_hosted_question"
+
+	stepTitleSelfHostedQuestion = ""
+
+	stepDescriptionSelfHostedQuestion = "Are you using a self-hosted private cloud or on-prem Zoom server?"
```

Comment:
Need a better way of wording this, it's a bit clunky at the moment. The message to get across is:

> Yes or no, do you have a private zoom instance?

Reply:
Are we sure Zoom offer exactly that? I would check Zoom's own wording on this.

Reply:
@DHaussermann What is your usual workflow for testing this setting?